bootstrap/src/main/java/se/fortnox/reactivewizard/Main.java

```diff
@@ -28,6 +30,11 @@ protected void configure() {
                 }
             };
             Guice.createInjector(new AutoBindModules(bootstrap));
+        } catch (CreationException e) {
+            for(Message message : e.getErrorMessages()) {
```

Comment:
Maybe add a unit test so that we can keep our overall coverage higher, we are soon at 90% :)

Reply:
Sure! I'll gather comments about the PR and add a test if the change makes sense in general.